test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -448,29 +448,38 @@ def __init__(
             ):
                 super().__init__()
                 self.conv = torch.nn.Conv2d(3, 128, kernel_size=3, stride=1)
+                self.conv2 = torch.nn.Conv2d(128, 128, kernel_size=3, stride=1)
 
             def forward(self, x):
-                return self.conv(x)
+                return self.conv2(self.conv(x))
 
         mod = M().eval()
         v = torch.randn((1, 3, 8, 8), dtype=torch.float32, requires_grad=False).add(1)
 
-        def matcher_check_fn():
-            # 1. Dequant-Conv2D pattern matched in QConv2D weight prepack * 1
-            #    [convert_element_type_1, sub, mul_1, dequantize_per_channel, clone, convolution]
-            self.assertEqual(
-                counters["inductor"]["qconv2d_weight_prepack_matcher_count"], 1
-            )
-            self.assertEqual(
-                counters["inductor"]["qconv2d_weight_prepack_matcher_nodes"], 6
-            )
+        for int8_mixed_bf16 in (
+            [False, True] if torch.ops.mkldnn._is_mkldnn_bf16_supported() else [False]
```

Comment:
will it be better to split this to two tests?

Reply:
Thanks for the suggestion and split.